src/hooks/useDrafts/useDrafts.tsx

```diff
@@ -27,9 +37,13 @@ type DraftState = {
   videos: VideoDraft[]
 }
 
+export type UnseenDrafts = { draftId: string; channelId: string }[]
```

Comment:
Let's use this as a singular type -`UnseenDraft`. We can do `UnseenDraft[]` when needed